bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx

```diff
@@ -20,17 +20,60 @@ categories:
+
+Upon further analysis, we found a design flaw in macOS 13 and 14, where brute-force attacks performed by malicious actors could result in the service halting. [Find out more about this issue on our blog](https://www.scaleway.com/en/blog/m1-how-a-design-flaw-slowed-down-apples-vnc/).
+
+As a precaution, every Mac mini comes with a preinstalled `fail2ban` service. For added security and to ensure VNC service reliability, you can restrict access to macOS Screen Sharing to certain static IPs. This can be done by tweaking your network settings and using macOS's inbuilt firewall, `pf` (Packet Filter), to manage incoming connections.
 
 <Message type="note">
   This is a known issue that multiple Apple users have reported over time. Unfortunately, Apple's support team has not effectively addressed it yet.
 </Message>
 
-If you are experiencing this issue, you can try the following solutions:
+## Enable and Configure pf (Packet Filter)
+
+### Turn on the firewall
+
+macOS comes with a built-in firewall called `pf`. If you haven't already, enable the macOS firewall by going to **System Settings** -> **Network** -> **Firewall**. Toggle the switch to **On** to enable it.
+
+### Configure the firewall
+
+You have to create a custom rule set for `pf` to only allow incoming Screen Sharing connections from your static IP.
 
-## Connect to the Mac mini M1 via SSH and restart Screen Sharing
+1. Open a **Terminal**.
+2. Backup the original `pf.conf` file by copying it to a different location.
+   ```
+   sudo cp /etc/pf.conf /etc/pf.conf.backup
+   ```
+3. Edit the `pf.conf` file with a command like `sudo nano /etc/pf.conf`.
+4. Add the following lines to write rules that block all incoming connections on the Screen Sharing port, except for the desired IP.
+    ```
+    scrub-anchor "com.apple/*"
+    nat-anchor "com.apple/*"
+    rdr-anchor "com.apple/*"
+    dummynet-anchor "com.apple/*"
+    anchor "com.apple/*"
+    load anchor "com.apple" from "/etc/pf.anchors/com.apple"
+    block drop in proto { tcp, udp } from <fail2ban> to any
+    ## Add these lines to block connections to port 5900 except from your static IP
+    pass in on en0 proto tcp from <static IP> to any port 5900
+    block in on en0 proto tcp from any to any port 5900
+    ```
+    In the example above, `en0` should be replaced with the network interface you are using (you may discover the name of your network interface name using the `ifconfig` command), and `<static IP>` should be the static IP address you are allowing. Port 5900 is typically used for VNC/screen sharing services.
+5. Save and close the file (in `nano`, it's **Control-X** to close, **Y** to confirm changes, and **Enter** to save).
+6. Run `sudo pfctl -e -f /etc/pf.conf` to load the new `pf` configuration. You might need to restart your computer for changes to take effect.
```

Comment:
pf should be already enabled so the `-e` shouldn't be needed I guess?

also, if the a restart is needed then it is not the right command line.